src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonParameterInfo.cs

```diff
@@ -20,7 +20,7 @@ internal abstract class JsonParameterInfo
         // The default value of the parameter. This is `DefaultValue` of the `ParameterInfo`, if specified, or the CLR `default` for the `ParameterType`.
         public object? DefaultValue { get; private protected set; }
 
-        public bool IgnoreDefaultValuesOnRead { get; private set; }
+        public bool IgnoreNullTokensOnRead { get; private set; }
```

Comment:
do we need to rename this?

Reply:
The previous name was inaccurate, this settings specifically concerns ignoring `JsonTokenType.Null` rather than "default" values